pom.xml

```diff
@@ -71,6 +71,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data-text</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.windup.rules</groupId>
+            <artifactId>windup-rulesets</artifactId>
+            <scope>provided</scope>
```

Comment:
I had to add a line here to get the project to build:

`<version>${version.windup-rulesets}</version>`